app/(tabs)/squad.tsx

```diff
@@ -45,76 +47,76 @@ export default function Squad() {
           <Text style={styles.text}>Forwards ({clubPlayers?.positionCount.forward})</Text>
           <View style={styles.wrapContainer}>
             {forwards.length ? 
-                forwards.map((player) => {
-                    return (
-                        <CardLayout style={styles.card}>
-                            <View>
-                                <Text style={styles.text}>{player.name}</Text>
-                                <Text style={styles.playerDetails}>Goals: {player.goals}</Text>
-                                <Text style={styles.playerDetails}>Assists: {player.assists}</Text>
-                                <Text style={styles.playerDetails}>Average rating: {player.ratingAve}</Text>
-                            </View>
-                        </CardLayout>
-                    )
-                }) : 
-                <Text style={styles.playerDetails}>No forwards in the club</Text>
+              forwards.map((player) => {
```

Comment:
This seems to be very repetitive as you are setting the same children for  pretty much all the field positions. Can you figure out a way to reuse the logic?

Reply:
Done!
Will be changed in the next PR.